packages/@apphosting/common/src/index.ts

```diff
@@ -1,56 +1,57 @@
 import { spawn } from "child_process";
 
-// Output bundle specifications to be written to bundle.yaml
-export interface OutputBundle {
-  version: "v1alpha";
+// Output bundle metadata specifications to be written to bundle.yaml
+export interface OutputBundleConfig {
+  version: "v1";
   serverConfig: ServerConfig;
   metadata: Metadata;
 }
 
 // Fields needed to configure the App Hosting server
 interface ServerConfig {
-  // Command to start the server (e.g. ["node", "dist/index.js"]) assume this command is run from the root dir of the workspace
-  runCommand: string[];
-  // Environment variable present in the server execution environment.
+  // Command to start the server (e.g. "node dist/index.js"). Assume this command is run from the root dir of the workspace
+  runCommand: string;
+  // Environment variables set when the app is run
   environmentVariables?: EnvVarConfig[];
+  // See https://firebase.google.com/docs/reference/apphosting/rest/v1beta/projects.locations.backends.builds#runconfig for documentation on the next fields
   // The maximum number of concurrent requests that each server instance can receive.
-  // Defaults to 80.
   concurrency?: number;
   // The number of CPUs used in a single server instance.
-  // Defaults to 1.
-  cpu?: "fractional" | 1 | 2 | 4 | 6 | 8;
+  cpu?: number;
   // The amount of memory available for a server instance.
-  // Commonly one of: "256" | "512" | "1024" | "2048" | "4096" | "8192" | "16384"
-  // Defaults to 512MiB.
   memoryMiB?: string;
```

Comment:
why string?

Reply:
Good catch, we probably want number here too